### DISCUSSION TITLE
Gauges and Signals controls

### DISCUSSION THREAD

**Discussion by Waxime64** [MahApps/MahApps.Metro#4204] [Ideas]
Hi,
I like your controls, but they missing only 2 types. 
Gauge: For show with arrow a value. 
Signal: Indicator where you are in range. Exemple Wifi signal

Thanks

**Comment by timunie** [comment]
Hi @Waxime64 

glad you like MahApps :-) . And yes of course, there are thousands of ideas for MahApps. But as it is a free time project, it may take long time to implement, if even someone finds the time. I hope for your understanding. 

Happy coding 
Tim
